src/redux/books/book.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
- [ ] Kindly, move this component along with the `books` component to the components directory since this is a sub folder of the redux directory. Files in the redux directory should only be responsible for store interaction such as adding/removing books using reducers(will be covered in a later milestone).

Reply:
Thank you for this suggestion! This helped me understand the file structure needed.